spec/exceptions.md

```diff
@@ -55,4 +55,4 @@ The following exceptions are thrown by certain C\# operations.
   System.TypeInitializationException   Thrown when a static constructor or static field initializer throws an exception, and no catch clause exists to catch it.
   ------------------------------------ ------------------------------------------------------------------------------------------------------------------------------------------
 
-[[[]{#_Ref456607421 .anchor}]{#_Toc445783092 .anchor}]{#_Toc456601364 .anchor}
```

Comment:
This change is great. However, as I look through this file, there are a number of class names that aren't code-fenced. Can you make those updates?